src/pages/Account/Login.tsx

```diff
@@ -38,9 +39,15 @@ export const Login: React.FC<LoginProps> = ({
 
   const [formToDisplay, setFormToDisplay] = useState(formType);
 
-  const { signInWithSAML, hasSamlAuthProviderId } = useSAMLSignIn(
-    venue.samlAuthProviderId
-  );
+  const {
+    isShown: isLoginLoadingShown,
+    // show: showLoginLoading,
+    // hide: hideLoginLoading,
+  } = useShowHide(false);
+
+  const { signInWithSAML, hasSamlAuthProviderId } = useSAMLSignIn({
+    samlAuthProviderId: venue.samlAuthProviderId,
+  });
```

Comment:
What's the need/point of `useShowHide` here now? Use one of the `useAsync` variants within `useSAMLSignIn` instead.

Reply:
Done in https://github.com/sparkletown/sparkle/pull/1667/commits/36f9ee2f4cfc1f5342c9ef7c0843ba5452393d9f